plugins/schedule/base/lib/api/KalturaScheduleEvent.php

```diff
@@ -292,13 +307,33 @@ public function validateForInsert($propertiesToSkip = array())
 	{
 		$this->validatePropertyNotNull('recurrenceType');
 		$this->validatePropertyNotNull('summary');
-		$this->validatePropertyNotNull('startDate');
+		if (is_null($this->startDate) && is_null($this->linkedTo))
+		{
+			throw new KalturaAPIException(KalturaScheduleErrors::START_TIME_AND_LINKED_TO_CONFLICT);
+		}
+		if (!is_null($this->startDate) && !is_null($this->linkedTo))
+		{
+			throw new KalturaAPIException(KalturaScheduleErrors::START_TIME_AND_LINKED_TO_NOT_SET);
+		}

```

Comment:
The event here must that its a conflict and you cannot set both

Reply:
Done